src/pages/markets/components/markets/_synthetic-indices.js

```diff
@@ -36,10 +41,12 @@ const StockIndices = ({ simple_step_content }) => {
             <AvailableTrades
                 CFDs={<CFDs market_content={is_eu_country ? synthetic_cfds_eu : synthetic_cfds} />}
                 DigitalOptions={
-                    <DigitalOptions
-                        market_name={localize('synthetic indices')}
-                        options_list={synthetic_options}
-                    />
+                    !is_eu_country && (
+                        <DigitalOptions
+                            market_name={localize('synthetic indices')}
+                            options_list={synthetic_options}
+                        />
+                    )
```

Comment:
The old `is_eu_country` also have uk in its list.  If this content is related to new UK EU domains then we should use the new `is_eu_country` helper otherwise this will not show up in the UK as well. But if this is intended then we can keep it.